contracts/outlayer-app/README.md

```
@@ -0,0 +1,109 @@
+
+`near-op` computes the `StateInit` JSON for a new Outlayer App instance.
+
+### Running
+
+```sh
+cargo run -p defuse-outlayer-app --example near-op
+```
+
+### Usage
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)